someSong/src/pages/home/home.ts

```diff
@@ -34,6 +36,9 @@ export class HomePage {
       this._user.getUserQuestions(this.user.userID).on('child_added', userQuestion => {
         this._question.getQuestionDetails(userQuestion.key).subscribe((questionDetail) => {
           this.userQuestions = DictionaryHelpFunctions.addToDictionary(this.userQuestions, userQuestion.key, questionDetail);
+
+          this.questionLoading = false;
```

Comment:
You only change this value if there is a new question added by the logged in user.
If a user logs in and doesn't have any questions, the loading wheel keeps on spinning until a question is uploaded by the user.
I think u should check if the user doesn't have any questions yet, change the value of `this.questionLoading` to `false`.
Something like this - 
` if(!this.user.questions) {
        this.questionLoading = false;
      }`